featurebyte/models/feature_table_cache.py

```diff
@@ -0,0 +1,65 @@
+"""
+This module contains Feature Table Cache related models
+"""
+from typing import List
+
+import pymongo
+
+from featurebyte.models.base import (
+    FeatureByteBaseModel,
+    FeatureByteCatalogBaseDocumentModel,
+    PydanticObjectId,
+    UniqueConstraintResolutionSignature,
+    UniqueValuesConstraint,
+)
+
+
+class CachedFeatureDefinition(FeatureByteBaseModel):
+    """
+    Definition of the feature cached in Feature Table Cache
+    """
+
+    feature_id: PydanticObjectId
```

Comment:
If the feature is not saved yet e.g. an ideated feature, what should the feature_id be? Should this be optional?

Reply:
I wan thinking - ideated feature id (they are also saved into separate collection), but this will probably make it even more misleading. I will make this optional